tests/e2e/release-e2e-findings.md

```
@@ -0,0 +1,161 @@
+Branch: `main` @ `72a0e68` (after pulling `feat(server): add Anthropic
+Messages API ingress endpoint` #343 and `fix(providers): support OpenRouter
+virtual model IDs` #344).
+
+## Scope
+
+1. Ran the full existing release E2E matrix (`S01`–`S95`).
+2. Added 14 new scenarios (`S96`–`S109`) covering the new Anthropic Messages
+   API ingress endpoints (`POST /v1/messages`, `POST /v1/messages/count_tokens`)
+   and ran them.
+
+The runner only parsed 2-digit scenario IDs (`^### S[0-9][0-9] `). It was
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)